blueocean-web/src/main/js/try.js

```diff
@@ -0,0 +1,18 @@
+var $ = require('jquery-detached').getJQuery();
+var jsModules = require('@jenkins-cd/js-modules');
+
+$(document).ready(() => {
+    var tryBlueOcean = $('<div class="try-blueocean header-callout">Try Blue Ocean UI ...</div>');
```

Comment:
Fair enough ... I'll do that if I don't do the contextual thingie that I have in mind.